src/main/flight/imu.c

```diff
@@ -176,6 +175,7 @@ static float calculateThrottleAngleScale(uint16_t throttle_correction_angle)
 void imuConfigure(uint16_t throttle_correction_angle, uint8_t throttle_correction_value)
 {
     imuRuntimeConfig.imuDcmKp = imuConfig()->imu_dcm_kp / 10000.0f;
+    // current default for imu_dcm_kp is 2500, so this returns a base value for imuDcmKp of 0.25
```

Comment:
Can you put the comment at the top of the line, please? Comments are placed above the commented line or paragraph.

Reply:
ok